docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -65,7 +73,7 @@ storage systems:
 
 You must enable and configure the specific native file system access. If none is
 activated, the [legacy support](file-system-legacy) is used and must be
-configured.
+configured. However, legacy support will be deprecated and removed.
```

Comment:
"if none is activated, the legacy support is used"

This isn't true anymore. Instead, this should be a single sentence that reads "You must enable and configure the specific file system access"